pages/docs/plugins/grouparoo-calculated-property.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "@grouparoo/calculated-property"
+date: "2021-09-02"
+pullQuote: "Learn how to work with the Calculated Property plugin as a Source to generate custom transformed properties."
+pageNavSelector: "h2"
+---
+
+<PluginDocsLinks plugin="CalculatedProperty" />
+
+Grouparoo's Calculated Property plugin enables you to write a custom function to generate a new property based upon other existing Grouparoo Profile Properties.
+
+This guide will show you how to work with the Calculated Property plugin to [create a custom Calculated Property](#create-calculated-property-property)
+
+<PluginDocsInstallation plugin="CalculatedProperty" />
```

Comment:
the casing gets weird in the code generated here because the slug uses hyphens, but the install template uses snake case... it bothers me, but I'm wondering if it's ok to others?